.github/actions/session_nonzero/action.yml

```diff
@@ -0,0 +1,49 @@
+name: Azure Nonzero session execution
```

Comment:
I would rename the action to get straight to the point and not bother the "user" of the action with session 0 or none 0 (this could be in the description still)